src/JPEGView/SettingsProvider.h

```diff
@@ -131,6 +131,11 @@ class CSettingsProvider
 	bool BeepSoundAlert() { return m_bBeepSoundAlert; }
 	double ZoomPauseFactor() { return m_zoomPauseFactor; }  // while internally this is represented in doubles, using a whole number percent simplifies it for the user... configuring doubles is not user friendly at all
 
+	std::vector<double> ParseCustomZoomSteps(LPCTSTR data);

```

Comment:
Move this to the bottom around where `GetAutoZoomMode` is... `ParseCustomZoomSteps` should be private and grouped with those methods.